docs/ref/contrib/admin/index.txt

```diff
@@ -1230,6 +1230,14 @@ subclass::
         default search method but uses an index. Currently this is only
         available for MySQL.
 
+    ``!``
+        Use the '!' operator to specify your own field lookup. For example
+        if ``search_fields`` is set to ``['!first_name__exact']`` and
+        user searches for ``John Lennon``, Django will do the equivalent
+        of this SQL ``WHERE`` clause::
+
+            WHERE (first_name = 'John Lennon')
+
```

Comment:
Perhaps how this allows avoiding the concerns in https://code.djangoproject.com/ticket/26001 could also be documented.